dropshot/src/test_util.rs

```diff
@@ -62,6 +62,7 @@ const ALLOWED_HEADERS: [AllowedHeader<'static>; 5] = [
         name: "transfer-encoding",
         value: AllowedValue::OneOf(&["chunked"]),
     },
+    AllowedHeader::new("x-dropshot-test-header"),
```

Comment:
nit: since this is one we know we use internally, it'd be sad if we had a bug that leaked this header and we didn't detect it in this test.

I guess the header itself is only ever used in the tests so maybe that's not much of a risk.  The alternative seems like it would be to allow each to test to configure the allowable headers, which might be kind of nice, but seems like a lot of work to take on for this case.

Reply:
It seemed like more work than was warranted to let each test tack on additional allowed headers, but I let me know what you think. It's probably not **that** much more work.

Reply:
Having the header name be a static in the tests is already a help.